systemtest/src/main/java/io/strimzi/systemtest/utils/ClientUtils.java

```diff
@@ -236,15 +240,28 @@ public static String generateRandomConsumerGroup() {
         return CONSUMER_GROUP_NAME + salt;
     }
 
-    public static KafkaClientsBuilder getDefaultClientBuilder(TestStorage testStorage) {
+    public static KafkaClientsBuilder getDefaultBasicClientBuilder(TestStorage testStorage) {
```

Comment:
default basic doesn't say much about the clients inside
it's basic because:
- it uses things from TestStorage?
- it uses plain bootstrap address?
- it sets completely everything?

I would rename it to something more relevant, when you have continuous part, you can have immediate or something.
Also, I would remove "default" and added something like `fromTestStorage` or something like that.

And finally, I know that it's there for a long time, but I don't really like the idea of setting everything - f.e. username - if it will not be used.

You can do for example this:

Add few methods inside the builder classes, with name f.e. "tlsClientsBasedOnTestStorage" and there you can set whatever you like, but for TLS.

Because if you use  `ClientsUtils.getDefaultBasicClientBuilder(testStorage)` and multiple places across the whole suite, I'm pretty sure that in few months, you will have to take a look what's set there before using it in different test.